### PR TITLE
Rufus hamade deployment work

### PR DESCRIPTION
**Pull request by RufusHamade**
Based on RufusHamade-maintenance-fixes-1, 

Contains the docs and scripts required to deply OSMod to a kubernetes cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,12 +93,123 @@ You'll also need to create some (human) users.  The email address of the new use
...
+  "ATTACK_ON_COMMENTER": {},
+  "INCOHERENT": {},
+  "INFLAMMATORY": {},
+  "OBSCENE": {},
+  "OFF_TOPIC": {},
+  "SPAM": {} ,
+  "UNSUBSTANTIAL": {},
+  "LIKELY_TO_REJECT": {},
+  "TOXICITY": {},
+  "TOXICITY_FAST": {}
```
Comment (iislucas) [inline README.md]: Lets remove toxicity_fast. It's not a good model.

File: README.md [hunk 2]
```diff
@@ -66,12 +93,123 @@ You'll also need to create some (human) users.  The email address of the new use
...
+}
+EOM
+
+cd $PERSPECTIVEAPI_PROXY_LOCATION
+yarn install
+PORT=8081 ATTRIBUTE_REQUESTS='{ "TOXICITY": {} }' yarn run watch
+```
+
+TODO: Need fixes to proxy and osmod:
+ATTACK_ON_PUBLISHER is in proxy's default list and osmod UI, but doesn't work.
```
Comment (iislucas) [inline README.md]: Yeah, remove it from the list for now.

File: README.md [hunk 3]
```diff
@@ -66,12 +93,123 @@ You'll also need to create some (human) users.  The email address of the new use
...
+SUMMARY_SCORE in osmod UI, but doesn't work.
```
Comment (iislucas) [inline README.md]: summary score should be a field inside each attribute's response, not an attribute in its own right.

File: deployments/local/Dockerfile [hunk 4]
```diff
@@ -1,11 +1,14 @@
 FROM gcr.io/google_appengine/nodejs
 
-RUN install_node v6.11.0
+RUN install_node v8.11.1
 
+WORKDIR /app/
 COPY . /app/
 
-RUN npm install --unsafe-perm
+RUN npm cache verify && find . -name node_modules -exec rm -Rf {} \; && find . -name package-lock.json -exec rm -Rf {} \;
```
Comment (iislucas) [inline deployments/local/Dockerfile]: Can you add a comment for why/what you need the find things for. This seems a bit surprising to me.

**Comment by iislucas** [comment]
There's still some pending comments on the PR... are you planning to do more... ? Just wanting to check I'm not accidentally blocking you in a code review?

**Comment by RufusHamade** [comment]
Hi Lucas,

I'm just implementing your suggestions.  Actually I have implemented them already - now just testing....

Thanks very much, I really appreciate this feedback, especially as I'm still learning the codebase.

File: bin/run [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd packages/backend-api
+PORT=8080 node ./server.js &
+cd -
+
+sleep 5
```
[the remark this answers is not in the document]
Comment (RufusHamade) [reply bin/run]: Sleep was not actually necessary.  I just copied it from the "watch" script, where it *is* necessary.  I removed it now.

File: deployments/local/Dockerfile (hunk 4 above)
Comment (RufusHamade) [reply deployments/local/Dockerfile]: Turns out, it wasn't necessary.  Image build issues were actually caused by bad internet connection at office where I did the development.  Running the scripts at home makes it all just work.

I removed the unnecessary code.

**Comment by RufusHamade** [comment]
Hi Lucas,

Uploaded a patch with the review markups.  The patch is non-trivial as I've also patched the initial-database.sql script to create the "tags" entries as discussed above, i.e.,
- removed "Attack on publisher"
- set inSummaryScore for all tags.
Also reordered things a bit and removed gaps.  

This means I could remove all the TODOs from the README.

If you have no more comments, then I think we can land this change?

File: README.md [hunk 6]
```diff
@@ -66,12 +93,116 @@ You'll also need to create some (human) users.  The email address of the new use
...
 ```bash
-./bin/osmod ...
+./bin/osmod users:create --group service --name "PerspectiveProxy" --endpoint=<proxy URL>
 ```
 
+where `<proxy URL>` is the URL of [The Perspective API proxy] you plan on using.
+
+### The Perspective API proxy
+
+OSMod is designed to talk to the Perspective API via a proxy server: the
```
Comment (iislucas) [inline README.md]: Maybe note also that we plan to remove this and package the proxy with Omod?

Comment (RufusHamade) [reply README.md]: Actually, I decided not to eliminate the proxy entirely, as that is what NYT is currently using, and I didn't want to mess them around too much.  It is now configurable.  I'll queue up that change for review next.

That change rewrites this text fragment a little bit.  So we can update it as you see fit in that review.

**Pull request merged**